ui.apps/src/main/content/jcr_root/etc/acs-commons/lists/.content.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="sling:OrderedFolder">
+    <font-awesome-icons/>
```

Comment:
@DeepaliAgrwl : why do we have other list here ?

Reply:
removed the extra lists from code.